packages/pangea-sdk/PangeaCyber.Net/Redact/Models/PartialMasking.cs

```diff
@@ -0,0 +1,24 @@
+namespace PangeaCyber.Net.Redact
+{
+    ///
+    public class PartialMasking
+    {
+        ///
+        public MaskingType? MaskingType { get; set; } = default!;
+        ///
+        public int? UnmaskedFromLeft { get; set; } = default!;
+        ///
+        public int? UnmaskedFromRight { get; set; } = default!;
+        ///
+        public int? MaskedFromLeft { get; set; } = default!;
+        ///
+        public int? MaskedFromRight { get; set; } = default!;
+        ///
+        public List<string>? CharsToIgnore { get; set; } = default!;
+        ///
+        public List<string>? MaskingChar { get; set; } = default!;
```

Comment:
Can these two lists be of `char`s instead?

Reply:
Hmm, maybe yes, I guess `char`s would be serialized as strings, isn't it?